packages/repack/android/src/newarch/java/com/callstack/repack/ChunkManagerModule.kt

```diff
@@ -0,0 +1,25 @@
+package com.callstack.repack
+
+import android.os.Handler
+import com.facebook.react.bridge.*
+
+class ScriptManagerModule(reactContext: ReactApplicationContext) : NativeScriptManagerSpec(reactContext) {
+    private val remoteLoader: RemoteScriptLoader = RemoteScriptLoader(reactApplicationContext)
+    private val fileSystemLoader: FileSystemScriptLoader = FileSystemScriptLoader(reactApplicationContext)
+
+    private var implementation: ScriptManagerModuleImpl = ScriptManagerModuleImpl()
+
+    override fun getName(): String = ScriptManagerModuleImpl.NAME
+
+    override fun loadScript(scriptId: String, configMap: ReadableMap, promise: Promise) {
```

Comment:
Could you check, if this method is called on the TS side for the new arch? AFAIR `@ReactMethod` is also needed for the new arch as well.

Reply:
AFAIK, `@ReactMethod` isn't needed for the newArch, it instead uses the `override`